store/src/app/products/components/header/header.component.html

```diff
@@ -0,0 +1,16 @@
+<header class="header">
+  <nav class="header__nav">
+    <ul class="header__list">
+      <li class="header__item">
+        <img class="main-logo">
+      </li>
+      <li class="header__item">
+        <a class="header__item-link" routerLink="/products">Products</a>
```

Comment:
Use [routerLinkActive](https://angular.io/api/router/RouterLinkActive) to highlight active page

Reply:
done